python/tvm/relay/op/contrib/arm_compute_lib.py

```diff
@@ -111,20 +111,38 @@ def convert_conv(attrs, inputs, tinfos, desired_layouts):
 
         return convert_conv
 
-    with TempOpAttr(
-        "nn.conv2d", "FTVMConvertOpLayout", convert_layout_conv2d(tvm.relay.nn.conv2d)
-    ), TempOpAttr(
-        "qnn.conv2d", "FTVMConvertOpLayout", convert_layout_conv2d(tvm.relay.qnn.op.conv2d)
-    ):
-        seq = tvm.transform.Sequential(
-            [
-                transform.ConvertLayout(
-                    {"nn.conv2d": ["NHWC", "OHWI"], "qnn.conv2d": ["NHWC", "OHWI"]}
-                ),
-                transform.FoldConstant(),
-            ]
-        )
-        preprocessed_mod = seq(mod)
+    # Set tmp attributes
+    attr_key = "FTVMConvertOpLayout"
+
+    op_nn = relay.op.get("nn.conv2d")
+    older_attr_nn = op_nn.get_attr(attr_key)
+    op_nn.reset_attr(attr_key)
+    op_nn.set_attr(attr_key, convert_layout_conv2d(tvm.relay.nn.conv2d))
+
+    op_qnn = relay.op.get("qnn.conv2d")
+    older_attr_qnn = op_qnn.get_attr(attr_key)
+    op_qnn.reset_attr(attr_key)
+    op_qnn.set_attr(attr_key, convert_layout_conv2d(tvm.relay.qnn.op.conv2d))
```

Comment:
Better to use `with` context as `TempOpAttr` does to make sure the changes won't affect other processes. I think it's fine to just copy `TempOpAttr` here to be `ArmComputeLibOpAttrContext` or something like that.

Reply:
Agreed, I think this would make it more readable as well, otherwise this looks good to me!

Reply:
I have added `OpAttrContext` (which is a copy of `TempOpAttr`) in `arm_compute_lib.py`.
And I have replaced `TempOpAttr` with `OpAttrContext`.